### ISSUE TITLE
How to use the FixRes method ?

### ISSUE THREAD

**Issue by MTfast** [facebookresearch/FixRes#22]
Thanks for your great work, I have a question on how to use the FixRes method to improve the performance of CNN and where is it used in the code ?

**Comment by TouvronHugo** [comment]
Hi @MTfast ,
FixRes consists of finetune the fully connected layer and the last batch norm , with higher resolution and validation data augmentation.
The data augmentation can be found in the file: transforms_v2.py.
The finetuning can be found in the file: imnet_finetune/train.py.
I hope I answered all your questions, don't hesitate if you have any more questions.

Best,

Hugo
